src/main/app/src/components/toteutus/ToteutusHakuMuu.js

```diff
@@ -72,12 +72,17 @@ export const ToteutusHakuMuu = ({ oid }) => {
               <Grid item container direction="row">
                 <Grid item xs md={4}>
                   <Typography noWrap variant="body1">
-                    {t('koulutus.hakuaika') + ':'}
+                    {t(
+                      muuHaku.hakuaika?.paattyy
+                        ? 'koulutus.hakuaika'
+                        : 'toteutus.haku-alkaa'
+                    ) + ':'}
```

Comment:
Kuuluisiko kaksoispiste suoraan käännökseen? eli siis translation.jsoniin

Reply:
Kyllä se on parempi käytäntö, laitan sinne

Reply:
Täällä oli paljon hyvää kaksoispisteen sisällyttämisestä suoraan käännökseen:
https://softwareengineering.stackexchange.com/questions/234546/colons-in-internationalized-ui

* Kaksoispiste toimii ihan eri tavalla eri kielillä (konfon tapauksessa tosin fi-sv-en käyttää kaksoispistettä samalla tavalla)
* Kääntäjän työ helpottuu koska konteksti on selkeämpi

Reply:
ok